src/main/java/openfasttrack/importer/specobject/ImportHelper.java

```diff
@@ -55,36 +67,73 @@ void runImport() throws XMLStreamException
 
             if (currentEvent.isStartElement())
             {
+                if (skipFile(currentEvent.asStartElement()))
+                {
+                    return;
+                }
                 foundStartElement(currentEvent.asStartElement());
             }
         }
     }
 
+    private boolean skipFile(final StartElement startElement)
```

Comment:
The name "skipFile" does not fit. No file is skipped in this method. Instead this method checks the conditions that lead to skipping a file in another method.

Suggested: "isXmlStartWhitelisted()"